### PR TITLE
Org serializer link update

### PR DESCRIPTION
**Pull request by zwolf**
@camallen added the specs, I made "org" say "organization". This fixes the broken href in serialized orgs. 

Fixes https://github.com/zooniverse/Panoptes/issues/2586

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

**Comment by zwolf** [comment]
Unless you're extremely worried about a couple (tops) of lingering rows in the media table, this is new and has never been used outside of a bit of testing and no data migration is required.

**Comment by camallen** [comment]
I'd prefer not to leave data out of sync with code / schema. Prod has 0, staging has 23 of these media resources. I'm not worried about staging and I'll leave it up to you, though a migration should be as easy as:
`Medium.where(type: "org_attached_image").update_all(type: "organization_attached_image")`

**Comment by zwolf** [comment]
That should do it then, yeah?

File: db/migrate/20180115214144_add_announcement_to_organization_contents.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class AddAnnouncementToOrganizationContents < ActiveRecord::Migration
   def change
     add_column :organization_contents, :announcement, :string, default: ""
+    Medium.where(type: "org_attached_image").update_all(type: "organization_attached_image")
```
Comment (camallen) [inline db/migrate/20180115214144_add_announcement_to_organization_contents.rb]: Hasn't this migration already run on staging? so this change won't do anything...

Comment (zwolf) [reply db/migrate/20180115214144_add_announcement_to_organization_contents.rb]: Has it? This PR hasn't merged yet, why would it have deployed to staging?

Comment (zwolf) [reply db/migrate/20180115214144_add_announcement_to_organization_contents.rb]: yeah, but a whole extra migration just for this seems like overkill. I'll fix it on the console.

Comment (camallen) [reply db/migrate/20180115214144_add_announcement_to_organization_contents.rb]: 👍
